packages/unmock-core/src/index.ts

```diff
@@ -1,24 +1,25 @@
 // Sinon for asserts and matchers
-import * as fetch from "node-fetch";
 import * as sinon from "sinon";
-import Backend from "./backend";
-import NodeBackend from "./backend/node";
+import Backend, { buildRequestHandler } from "./backend";
 import { ILogger, IUnmockOptions, IUnmockPackage } from "./interfaces";
-import FsSnapshotter, { ISnapshot } from "./loggers/snapshotter";
-import WinstonLogger from "./loggers/winston-logger";
+import { ISnapshot } from "./interfaces";
 import { ExtendedJSONSchema, nockify, vanillaJSONSchemify } from "./nock";
 import internalRunner, { IRunnerOptions } from "./runner";
+import { IService } from "./service/interfaces";
 import { AllowedHosts, BooleanSetting } from "./settings";
 
-export { fetch };
+export * from "./interfaces";
```

Comment:
This is bad, maybe we like to create `unmock-types` just containing the core types so that libraries like `unmock-fetch` can depend only on that.

Reply:
Agreed, even exporting it as `unmock-core/interfaces` as a first step would be cleaner.
Definitely for a different PR though...